src/mono/netcore/sample/iOS/Plist.in

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en-US</string>
+	<key>CFBundleExecutable</key>
+	<string>HelloiOS</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.dotnet.HelloiOS</string>
```

Comment:
```suggestion
	<string>net.dot.HelloiOS</string>
```

not too pretty but it'd be better aligned with the convention of using your domain name (dot.net)